vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -29,12 +29,13 @@ import (
 	"k8s.io/klog"
 )
 
-const (
+var (
 	// Ignore change priority that is smaller than 10%.
-	defaultUpdateThreshold = 0.10
+	defaultUpdateThreshold = flag.Float64("priority-update-threshold", 0.10, "Ignore change priority that is smaller this value")
+
 	// Pods that live for at least that long can be evicted even if their
 	// request is within the [MinRecommended...MaxRecommended] range.
-	podLifetimeUpdateThreshold = time.Hour * 12
+	podLifetimeUpdateThreshold = flag.Duration("priority-lifetime-threshold", time.Hour*12, "Pods that live for at least that long can be evicted even if their request is within the [MinRecommended...MaxRecommended] range")
```

Comment:
I think current name of this flag could be more descriptive. What do you think about `in-recommendation-bounds-eviction-lifetime-threshold`?

Reply:
I think prefix `in-recommendation` is unnecessary

what do you think about this names with pod prefix?

| Name | Default | Usage |
| --- | --- | --- |
| `pod-update-threshold` | 0.1 | Ignore updates that have priority lower than the value of this flag |
| `pod-eviction-lifetime-threshold` | 12h | Pods that live for at least that long can be evicted even if their request is within the [MinRecommended...MaxRecommended] range |

Reply:
I think that the `in-recommendatiotion-bounds` part is necessary (we will evict younger pods, if they are out of the [MinRecommended...MaxRecommended]  interval). 

`pod-eviction-lifetime-threshold` sounds like we wouldn't evict younger pods no matter what to me

Reply:
ok - finally we got this flags. Ok?

| Name | Default | Usage |
| --- | --- | --- |
| `in-recommendation-update-threshold` | 0.1 | Ignore updates that have priority lower than the value of this flag |
| `in-recommendation-bounds-eviction-lifetime-threshold` | 12h | Pods that live for at least that long can be evicted even if their request is within the [MinRecommended...MaxRecommended] range |

Reply:
Sounds good. Please update the PR

Reply:
PR updated

Reply:
Looks like I didn't explain myself clearly enough, sorry about that. I think that we need the `in-recommendation-bounds` prefix for the flag that controls after how long VPA will evict pods which have requests in the  `\[`MinRecommended`...`MaxRecommended`] interval.

We don't need the prefix for the flag which controls priority threshold, below which we will ignore updates.


So I'm suggesting

| Name | Default | Usage |
| --- | --- | --- |
| pod-update-threshold | 0.1 | Ignore updates that have priority lower than the value of this flag |
| in-recommendation-bounds-eviction-lifetime-threshold | 12h | Pods that live for at least that long can be evicted even if their request is within the [MinRecommended...MaxRecommended] range|

Reply:
LGTM - PR updated